styles/components/_modeRules.scss

```diff
@@ -0,0 +1,24 @@
+.rules {
+  padding: 1.25rem;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
```

Comment:
We must declare variable of this box-shadow in _variables.scss, because I see this value often.

Reply:
I'll change that in another PR together with some other fixes.